java/Ice/config/settings.gradle.kts

```diff
@@ -3,13 +3,19 @@
 pluginManagement {
     repositories {
         mavenLocal()
-        maven {
-            url = uri("https://download.zeroc.com/nexus/repository/maven-nightly/")
-        }
+        maven("https://download.zeroc.com/nexus/repository/maven-nightly/")
         gradlePluginPortal() // Keep this to allow fetching other plugins
     }
 }
 
-rootProject.name = "greeter"
+dependencyResolutionManagement {
+    repositories {
+        // This demo uses the latest Ice nightly build published in ZeroC's maven-nightly repository.
+        maven("https://download.zeroc.com/nexus/repository/maven-nightly/")
+        mavenCentral()
+    }
+}
+
+rootProject.name = "config"
```

Comment:
Some of the project names were copy/pasted without ever being updated.
This is the `config` demo`. Not the `greeter` demo, from which it was copied.